core/lib/dal/src/eth_sender_dal.rs

```diff
@@ -56,6 +56,55 @@ impl EthSenderDal<'_, '_> {
         Ok(txs.into_iter().map(|tx| tx.into()).collect())
     }
 
+    pub async fn get_eth_txs_history_entries_max_id(&mut self) -> usize {
```

Comment:
nit: I think you should somehow mark `get_eth_txs_history_entries_max_id` and  `get_last_sent_eth_tx_hash` as methods that should be used only for tests, if I'm not mistaken we just do a separate `impl {}` block in other dals for this purpose with a comment "use these methods only for tests"

Reply:
nice! I've moved it to a separate impl